lib/api/github.js

```diff
@@ -59,8 +59,10 @@ async function initRepo(repoName, token, endpoint) {
 async function findFilePaths(fileName) {
   const res = await ghGot(`search/code?q=repo:${config.repoName}+filename:${fileName}`);
   const exactMatches = res.body.items.filter(item => item.name === fileName);
-  const filePaths = exactMatches.map(item => item.path);
-  return filePaths;
+
+  // GitHub seems to return files in the root with a leading `/`
+  // which then breaks things later on down the line
+  return exactMatches.map(item => item.path.replace(/^\//, ''));
 }
```

Comment:
I am using Github Enterpise (v3 API) and I was seeing a root level package.json file being returned as `/package.json`. I wouldn't have thought anything of it but all other package.json files (e.g. ones not in the root) were returned like `path/to/package.json`.

Later down the line if `/package.json` was passed as `filePath` to `createTree` the GitHub API was returning a `422 Unprocessable Entity` response.

Not really sure what is going on 😕